.github/workflows/manylinux_2014.yml

```diff
@@ -148,16 +155,18 @@ jobs:
             -w /work/src \
             ${{ fromJSON(needs.docker.outputs.images).ov_build.manylinux2014_x86_64 }} \
             /bin/bash -c "
-              cmake -DENABLE_CPPLINT=OFF -DENABLE_NCC_STYLE=OFF -DCMAKE_VERBOSE_MAKEFILE=ON -DENABLE_PYTHON=OFF -DENABLE_WHEEL=OFF -S /work/src -B /work/build &&
+              python3.12 -m pip install -r /work/src/src/bindings/python/wheel/requirements-dev.txt
+              cmake -DPython3_EXECUTABLE=/usr/local/bin/python3.12 -DENABLE_CPPLINT=OFF -DENABLE_NCC_STYLE=OFF -DENABLE_TESTS=ON -DCMAKE_VERBOSE_MAKEFILE=ON -DENABLE_OV_TF_FRONTEND=OFF -DENABLE_OV_TF_LITE_FRONTEND=OFF -DENABLE_OV_PADDLE_FRONTEND=OFF -DENABLE_OV_PYTORCH_FRONTEND=ON -DENABLE_OV_JAX_FRONTEND=OFF -DENABLE_OV_ONNX_FRONTEND=ON -DENABLE_PYTHON=ON -DENABLE_WHEEL=ON -S /work/src -B /work/build &&
```

Comment:
Have to use one of the Pythons to build the tests once.

Reply:
All these: 
![image](https://github.com/user-attachments/assets/d55ecb23-8bbf-4864-8d00-3d0d6c3a2ab8)
+ all the `tests`:
![image](https://github.com/user-attachments/assets/73362e46-8956-4751-8a57-00e9ec7d8188)
I guess we do not need all of them in the archive passing down to API jobs.